docs/deployment/cumulus-distribution.md

```diff
@@ -0,0 +1,41 @@
+---
+id: cumulus_distribution
+title: Using the Cumulus Distribution API
+hide_title: false
+---
+
+The Cumulus Distribution API is a set of endpoints that can be used to enable AWS Cognito authentication when downloading data from S3.
+
+## Configuring a Cumulus Distribution deployment
+
+The Cumulus Distribution API is included in the main [Cumulus](https://github.com/nasa/cumulus/tree/master/tf-modules/cumulus_distribution) repo.
```

Comment:
Nit (and we aren't doing this elsewhere), we should probably explicitly call out the `.zip` target in the release documentation or tf-module README.   I've run into a few users not certain what this sort of statement means in terms of 'where do I get it from'.

Reply:
How about we add a note here in this deployment docs?

https://github.com/nasa/cumulus/pull/2353/commits/09f40077771b1e3886e58de81a9919a79eb39d99

And then we can also spell it out in the release notes?